src/block/block_vrfy.c

```diff
@@ -179,12 +179,14 @@ __verify_set_file_size(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_CKPT *ckpt)
  *     End file verification.
  */
 int
-__wt_block_verify_end(WT_SESSION_IMPL *session, WT_BLOCK *block)
+__wt_block_verify_end(WT_SESSION_IMPL *session, WT_BLOCK *block, int verify_error)
```

Comment:
Should `verify_error` be a boolean instead?

Reply:
I think `verify_error` would be better as an int since it tracks if we see an error code returned from verify. We could use a boolean with the name `verify_success`, depending on if we'd like to pass down the error code here.